Source/Core/InputCommon/CMakeLists.txt

```diff
@@ -178,30 +178,12 @@ if(UNIX)
   )
 endif()
 
-if(ENABLE_SDL)
-  find_package(SDL2)
-  if(SDL2_FOUND)
-    message(STATUS "Using shared SDL2")
-    set(SDL_TARGET SDL2::SDL2)
-  else()
-    # SDL2 not found, try SDL
-    find_package(SDL)
-    if(SDL_FOUND)
-      message(STATUS "Using shared SDL")
-      add_library(System_SDL INTERFACE)
-      target_include_directories(System_SDL INTERFACE ${SDL_INCLUDE_DIR})
-      target_link_libraries(System_SDL INTERFACE ${SDL_LIBRARY})
-      set(SDL_TARGET System_SDL)
-    endif()
-  endif()
-  if(SDL_TARGET AND TARGET ${SDL_TARGET})
-    target_sources(inputcommon PRIVATE
-      ControllerInterface/SDL/SDL.cpp
-      ControllerInterface/SDL/SDL.h
-    )
-    target_link_libraries(inputcommon PRIVATE ${SDL_TARGET})
-    target_compile_definitions(inputcommon PRIVATE "CIFACE_USE_SDL=1")
-  else()
-    message(STATUS "SDL NOT found, disabling SDL input")
-  endif()
+if(SDL2_FOUND)
+  target_sources(inputcommon PRIVATE
+    ControllerInterface/SDL/SDL.cpp
+    ControllerInterface/SDL/SDL.h
+  )
+  target_link_libraries(inputcommon PRIVATE SDL2::SDL2)
```

Comment:
This is ``SDL2::SDL2-static`` when using the version from Externals.

Reply:
I will leave cmake/macos changes for someone after this PR. If you change cmake to require higher version of sdl2 than brew (or whatever) provides, then the version in Externals should be preferred automatically, no?
I think theoretically the externals version can be built as .so , but static linking is probably easier, yes.